### ISSUE TITLE
Add a "Radio" feature

### ISSUE THREAD

**Issue by nagarev** [AurousGroup/Feature-Tracker#48]
I think It will be pretty awesome if you can add the option of "start a radio" streaming songs to your friends, or making a collaborative playlist in real time, this feature was awesome and really enjoyable in grooveshark. It helps to construct really strong bonds between people and the application.

Keep up the good work! :)

**Comment by srcnalt** [comment]
Well I am looking forward for this actually. That was the charm of Grooveshark, miss my station and friends there... :disappointed:

**Comment by nagarev** [comment]
Yes! It was the best. I'm feeling the same! Looking forward for this. :)

On Fri, Oct 16, 2015, 17:02 Sercan <notifications@github.com> wrote:

> Well I am looking forward for this actually. That was the charm of
> Grooveshark, miss my station and friends there... [image: :disappointed:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/AurousGroup/Feature-Tracker/issues/48#issuecomment-148820266>
> .
>
-- 
Nazaret
